src/main/java/core/basesyntax/operations/FruitTransaction.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.operations;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum Operation {
+        BALANCE("b"), SUPPLY("s"), PURCHASE("p"), RETURN("r");
```

Comment:
```suggestion
        BALANCE("b"), 
        SUPPLY("s"), 
        PURCHASE("p"), 
        RETURN("r");
```
Use this formatting for enum with values;)